src/neuro_relay/server.py

```
@@ -75,30 +79,44 @@ def submit_call_async_soon(self, cb, *args):
             loop.call_soon_threadsafe(cb, *args)
 
     # Called by the neuro-api when it wants to add ephemeral context
-    def add_context(self, game_title: str, message: str, reply_if_not_busy: bool):
+    def add_context(
+        self,
+        game_title: str,
+        message: str,
+        reply_if_not_busy: bool,
```

Ruff (0.14.8)

Boolean-typed positional argument in function definition

(FBT001)